RELEASE_INSTRUCTIONS.md

```diff
@@ -10,22 +12,21 @@ or a predetermined release date is approaching
 - [ ] Prepare the official release commit
   - [ ] Bump version in the version.txt
   - [ ] Move changes from the "Unreleased" section to the proper sections in the CHANGELOG.md
+  - [ ] Confirm all GitHub Actions tests pass
   - [ ] Change the draft to pull request to "ready for review"
   - [ ] Code Review with the Project Owner
   - [ ] When it is ready, merge into the master branch; this will generate a merge commit, and this commit will be the official release commit.
-- [ ] Confirm that this official release commit passes all continuous integration:
-  - [ ] [Build Plan](http://bamboo.corp.alleninstitute.org/browse/IFR-AAG)
 - [ ] Create a Release: https://github.com/AllenInstitute/ipfx/releases <"Draft a new release" button>
   - [ ] Create a draft release
-  - [ ] Summarize the release notes from the CHANGELOG.md and post them on on the Releases page on GitHub 
+  - [ ] Summarize the release notes from the CHANGELOG.md and post them on on the Releases page on GitHub
   - [ ] Review the release with the Project Owner
   - [ ] Publish the release
 
-### Publish:
+### PyPI and BKP
 
-- [ ] Bump the version in version.txt, if not bumped CI deployment to PyPi will fail
-- [ ] Set release version with "git tag v#.#.#" (e.g. "git tag v1.0.0", equivalent to the version you bumped to), this triggers circleci to publish ipfx to pypi
+- [ ] Reconfirm the version is correct in `version.txt`
+- [ ] Set release version with "git tag v#.#.#" (e.g. "git tag v1.0.0", equivalent to the version you bumped to), this triggers circleci to publish ipfx to PyPI (deprecated, likely need to move to GitHub Actions)
   - [ ] [Build](http://bamboo.corp.alleninstitute.org/deploy/viewDeploymentProjectEnvironments.action?id=164855841) and [Deploy](http://bamboo.corp.alleninstitute.org/deploy/viewDeploymentProjectEnvironments.action?id=164855841) IPFX Nightly
```

Comment:
Is the bamboo stuff still used/done?

Reply:
It wasn't. I updated the instructions to mention the specific commands for now.